### ISSUE TITLE
How to best load a static dataset inside the module?

### ISSUE THREAD

**Issue by achyutjoshi** [CJWorkbench/hello-workbench#2]
Hi,

Thank you so much for the quick responses. This is not an issue, but I could not find an appropriate tag.

I want to load a static dataset(about the US FIPS codes ) inside my module. The dataset would be used to match with the user's input data table. 

What are some good practices around that?

**Comment by adamhooper** [comment]
We have no pleasant way to do this. For now, the data needs to sit in the .py file.

I'll keep this issue open until we support loading static files in Python modules.

**Comment by achyutjoshi** [comment]
Thanks @adamhooper ! 

Please do correct me if I am wrong, by sitting in the .py file, you mean we define the data as a dictionary/list variable within the /py file?

**Comment by adamhooper** [comment]
Yes, your data can be variables within the .py file. For FIPS codes, dictionaries/lists should be okay.
